src/controllers/userController.ts

```diff
@@ -5,9 +5,19 @@ import userDB from '../models/user';
 class UserController {
 
     public async register(req: Request, res: Response) {
-        const { username, email, role, password, date , phone } = req.body;
-        const user = new userDB({username, email, password})
-        const resp = await user.save();
+        const { username, email,password, phone } = req.body;
+        
+        const user = new userDB({username, email, password , phone})
+        try {
+            const resp = await user.save();
+            res.json({
+                data: resp
+            })
+        } catch (error) {
+            console.log(error);
```

Comment:
please an error response to give information to the frontend side, otherwise, it will go to close the connection by timeout.